src/main/java/meteordevelopment/meteorclient/systems/modules/render/Blur.java

```diff
@@ -9,86 +9,62 @@
+        new Pair<>(3, 3.25),   // LVL 7
+        new Pair<>(3, 4.25),   // LVL 8
+        new Pair<>(3, 5.5),    // LVL 9
+        new Pair<>(4, 3.25),   // LVL 10
+        new Pair<>(4, 4.0),      // LVL 11
+        new Pair<>(4, 5.0),      // LVL 12
+        new Pair<>(4, 6.0),      // LVL 13
+        new Pair<>(4, 7.25),   // LVL 14
+        new Pair<>(4, 8.25),   // LVL 15
+        new Pair<>(5, 4.5),    // LVL 16
+        new Pair<>(5, 5.25),   // LVL 17
+        new Pair<>(5, 6.25),   // LVL 18
+        new Pair<>(5, 7.25),   // LVL 19
+        new Pair<>(5, 8.5)     // LVL 20
+    };
+
     // General
+    private final Setting<Integer> strength = sgGeneral.add(new IntSetting.Builder().name("strength").description("How strong the blur should be.").defaultValue(5).min(1).max(20).sliderRange(1, 20).build());
 
-    private final Setting<Mode> mode = sgGeneral.add(new EnumSetting.Builder<Mode>()
-        .name("mode")
-        .description("Which mode the blur should use.")
-        .defaultValue(Mode.Fancy)
-        .build()
-    );
-
-    private final Setting<Integer> radius = sgGeneral.add(new IntSetting.Builder()
-        .name("radius")
-        .description("How large the blur should be.")
-        .defaultValue(4)
-        .min(1)
-        .sliderRange(1, 32)
-        .build()
-    );
-
-    private final Setting<Integer> fadeTime = sgGeneral.add(new IntSetting.Builder()
-        .name("fade-time")
-        .description("How long the fade will last in milliseconds.")
-        .defaultValue(100)
-        .min(0)
-        .sliderMax(500)
-        .build()
-    );
+    private final Setting<Integer> fadeTime = sgGeneral.add(new IntSetting.Builder().name("fade-time").description("How long the fade will last in milliseconds.").defaultValue(100).min(0).sliderMax(500).build());
 
     // Screens
 
-    private final Setting<Boolean> meteor = sgScreens.add(new BoolSetting.Builder()
-        .name("meteor")
-        .description("Applies blur to Meteor screens.")
-        .defaultValue(true)
-        .build()
-    );
-
-    private final Setting<Boolean> inventories = sgScreens.add(new BoolSetting.Builder()
-        .name("inventories")
-        .description("Applies blur to inventory screens.")
-        .defaultValue(true)
-        .build()
-    );
-
-    private final Setting<Boolean> chat = sgScreens.add(new BoolSetting.Builder()
-        .name("chat")
-        .description("Applies blur when in chat.")
-        .defaultValue(false)
-        .build()
-    );
-
-    private final Setting<Boolean> other = sgScreens.add(new BoolSetting.Builder()
-        .name("other")
-        .description("Applies blur to all other screen types.")
-        .defaultValue(true)
-        .build()
-    );
-
-    private Shader shader;
-    private Framebuffer fbo1, fbo2;
+    private final Setting<Boolean> meteor = sgScreens.add(new BoolSetting.Builder().name("meteor").description("Applies blur to Meteor screens.").defaultValue(true).build());
```

Comment:
why did you remove the newlines for settings, every single other module has it this way and it is way more readable

Reply:
The reformatting feature of IntelliJ was used. My mistake, I will correct it.

Reply:
Done.

Reply:
> The reformatting feature of IntelliJ was used. My mistake, I will correct it.

I was hoping it was something like that and not a weird conscious decision, good job 👍🏼